lib/hummingbird.ex

```diff
@@ -69,20 +63,15 @@ defmodule Hummingbird do
       time: Event.now(),
       data: %{
         conn: Helpers.sanitize(conn),
-        caller: opts.caller,
+        name: opts.caller,
         traceId: conn.assigns[:trace_id],
         id: conn.assigns[:span_id],
         parentId: conn.assigns[:parent_id],
         user_id: conn.assigns[:current_user][:user_id],
         route: conn.assigns[:request_path],
         serviceName: opts.service_name,
-        name: opts.service_name
-        # when applicable, add durationMs
-        # ---
-        # Does not appear important in the honeycomb.ui, leaving off
-        #
-        # name: "http_request",
-        #
+        durationMs: conn.assigns[:request_duration],
```

Comment:
I swear to dog, if this starts populating duration.... I will be a happy man.  Unlocks most of the heatmaps I wanted to vis to show failures.